Use sjtc and generate /clock topic

fmauch opened the pull request:
With https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/1145 it is possible to use use the scaled JTC on systems that don't have the scaling state interface available. This way, we won't have to change the controller in the MoveIt configuration anymore in order to use the example MoveIt package with GZ Sim.

Draft until https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/1145 is merged.

fmauch committed:
Use scaled JTC by default

fmauch committed:
Add gz_bridge for /clock topic

fmauch committed:
Add dependency on ros_gz_bridge

VinDp reviewed (approved):
Looks good and works as intended.

fmauch committed:
Fix dependency and tests

Since we now use the scaled JTC we have to depend on ur_controllers
and make sure the tests use the scaled JTC.

fmauch commented:
Let's hold on that until the SJTC without required scaling interface is released and synced.

urfeex committed:
Merge branch 'ros2' into use_sjtc
